src/pages/vysledky.tsx

```diff
@@ -21,33 +47,24 @@ const Vysledky: NextPage<Props> = ({ taxForm }: Props) => {
   return (
     <>
       <Link href={previousRoute}>
-        <a className="govuk-back-link">Naspat</a>
+        <a className="govuk-back-link" data-test="back">
+          Naspat
+        </a>
       </Link>
-      <h2>Vysledky</h2>
-      <dl className="govuk-summary-list">
-        <div className="govuk-summary-list__row">
-          <dt className="govuk-summary-list__key">Základ dane</dt>
-          <dd className="govuk-summary-list__value">
-            {taxForm.r080_zaklad_dane_celkovo}
-          </dd>
-        </div>
-        <div className="govuk-summary-list__row">
-          <dt className="govuk-summary-list__key">Daň</dt>
-          <dd className="govuk-summary-list__value">{taxForm.r105_dan}</dd>
-        </div>
-        <div className="govuk-summary-list__row">
-          <dt className="govuk-summary-list__key">Daň na úhradu</dt>
-          <dd className="govuk-summary-list__value">
-            {taxForm.r125_dan_na_uhradu}
-          </dd>
-        </div>
-        <div className="govuk-summary-list__row">
-          <dt className="govuk-summary-list__key">Daňový preplatok</dt>
-          <dd className="govuk-summary-list__value">
-            {taxForm.r126_danovy_preplatok}
-          </dd>
-        </div>
-      </dl>
+      <h1>Výpočet dane za rok 2019</h1>
+      <h2>Stručný prehľad</h2>
+      <Summary rows={[
+        { title: 'Príjmy', value: taxForm.t1r2_prijmy },
+        { title: 'Zdravotné poistné', value: parseFloat(taxFormUserInput.priloha3_r13_zdravotne) },
```

Comment:
preco sa tu pouzive `taxFormUserInput` a vsade inde `taxForm`?

Reply:
Tato hodnota je iba v `taxFormUserInput`. Ak sa nemylim tak na danovom priznani nie su samostane polia pre zdravotku a socialku, ale iba suma pre vsetky poistenia. Kazdopadne ich nemame v type pre `taxForm`.

Reply:
TaxForm extenduje TaxFormUserInput cize tam je vsetko. Uznavam, ze to nemusi byt jasne na prvy pohlat a je to jemne krkolomne :)